src/frontend/js/modules/sidebar.js

```diff
@@ -32,6 +32,12 @@
 
     .directive('contextualSidebar', function($timeout, $window, $mdUtil, contextualSidebarService, session) {
       function link(scope, element, attr) {
+        // TODO: Write E2E test for this
+        // Because this directive scope changes behavior of a div (sidebar) outside of the button, so there is no way to unit test it
```

Comment:
@tuanlc : We can not unit test this directive to check if the search icon shows or hides because of this reason.